home/management/commands/translation_tracking.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
```

Comment:
Would it make more sense to move this file to `common/management/commands/translation_tracking.py`? Does that still work?

Reply:
Have tried this and it didn't work, but not really sure why, will read up a bit more on django to understand the management  function requirements better